course-materials/lab-instructions/lab-06/lab-06-ugly-charts.Rmd

```diff
@@ -48,35 +48,94 @@ use_git_config(user.name = "Your Name",
 
 This is the second week you're working in teams, so we're going to make things a little more interesting and let all of you make changes and push those changes to your team repository. Sometimes things will go swimmingly, and sometimes you'll run into merge conflicts. So our first task today is to walk you through a merge conflict!
 
-## Merge conflicts
 
-- When two collaborators make changes to a file and push the file to their repo, git merges these two files.
-- If these two files have conflicting content on the same line, git will produce a **merge conflict**.
+## Merges and merge conflicts
 
-### Set up
+- Pushing to a repo replaces the code on GitHub with the code you have on your computer. 
+- If a collaborator has made a change to your repo on GitHub that you haven't incorporated into your local work, GitHub will stop you from pushing to the repo because this could overwrite your collaborator's work!
+- So you need to explicitly "merge" your collaborator's work before you can push.
+- If your and your collaborator's changes are in different files or in different parts of the same file, git merges the work for you automatically when you *pull*.
+- If you both changed the same part of a file, git will produce a **merge conflict** because it doesn't know how which change to you want to keep and which change you want to overwrite.
```

Comment:
```suggestion
- If you both changed the same part of a file, git will produce a **merge conflict** because it doesn't know how which change you want to keep and which change you want to overwrite.
```